examples/with-elasticsearch/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't see eslint in the package.json, should this be removed?

Reply:
I think this got added while setting up `create-next-app`. Let me remove it!